test/e2e/k8sreporter/reporter.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package k8sreporter
+
+import (
+	"log"
+
+	performancev1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v1"
+	performancev1alpha1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v1alpha1"
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+
+	"github.com/openshift-kni/k8sreporter"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func New(kubeconfig, path, namespace string) *k8sreporter.KubernetesReporter {
+	// When using custom crds, we need to add them to the scheme
+	addToScheme := func(s *runtime.Scheme) error {
```

Comment:
What about machinconfig apis do we need them as well?

Reply:
yes, we need everything that is related to this operator. I'll add. thanks!